### ISSUE TITLE
[Feature]: Add TD3 to DRL_battery.py

### ISSUE THREAD

**Issue by manjavacas** [jajimer/sinergym#153]
### Feature 🚀

Include [TD3 ](https://stable-baselines3.readthedocs.io/en/master/modules/td3.html) to [DRL_battery.py](https://github.com/jajimer/sinergym/blob/main/DRL_battery.py)

### Motivation

It would be interesting to be added as it is an improved version of [DDPG](https://stable-baselines3.readthedocs.io/en/master/modules/ddpg.html).

### Solution

Add TD3 model to DRL_battery.py by following the reference constructor:

`TD3(policy, env, learning_rate=0.001, buffer_size=1000000, learning_starts=100, batch_size=100, tau=0.005, gamma=0.99, train_freq=(1, 'episode'), gradient_steps=- 1, action_noise=None, replay_buffer_class=None, replay_buffer_kwargs=None, optimize_memory_usage=False, policy_delay=2, target_policy_noise=0.2, target_noise_clip=0.5, tensorboard_log=None, create_eval_env=False, policy_kwargs=None, verbose=0, seed=None, device='auto', _init_setup_model=True)`

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/jajimer/sinergym/issues) in the repo (**required**)

:pencil: Please, don't forget to include more labels besides `Feature Request` if it is necessary.
